src/ontology/mondo-edit.obo

```diff
@@ -584963,6 +584963,18 @@ is_a: MONDO:0003767 {source="NCIT:C50888", source="https://orcid.org/0009-0009-6
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8003" xsd:anyURI
 
+[Term]
+id: MONDO:1030010
+name: precerebral artery stenosis
+def: "A disorder characterized by an abnormal narrowing of a precerebral artery." [https://orcid.org/0009-0009-6791-365X]
+synonym: "narrowing of precerebral artery" EXACT [SCTID:1055001]
+xref: ICD10CM:I65 {source="MONDO:relatedTo"}
+xref: SCTID:1055001 {source="MONDO:equivalentTo"}
+xref: UMLS:C0265089 {source="MONDO:equivalentTo"}
+is_a: MONDO:0011057 {source="https://orcid.org/0009-0009-6791-365X"} ! cerebrovascular disorder
```

Comment:
This orcid is private. Please add another source for this SC.

Reply:
updated